src/Pester.Runtime.ps1

```diff
@@ -687,7 +687,12 @@ function Invoke-TestItem {
                         Write-PesterDebugMessage -Scope Skip "($path) Test is skipped."
```

Comment:
Wrong for inconclusive, but mostly used for flow tracing, so we can leave it for now.